tidb-cloud/terraform-use-dedicated-network-container-resource.md

```diff
@@ -0,0 +1,229 @@
+
+    ```
+    terraform {
+      required_providers {
+        tidbcloud = {
+          source = "tidbcloud/tidbcloud"
+        }
+      }
+    }
+
+    provider "tidbcloud" {
+      public_key = "your_public_key"
+      private_key = "your_private_key"
+    }
+
+    resource "tidbcloud_dedicated_network_container" "example" {
+      project_id = "1372813089454000000"
+      region_id = "aws-ap-northeast-2"
+      cidr_notation = "172.16.16.0/21"
+    }
+    ```
+
+    Use the `resource` block to define the resource of TiDB Cloud, including the resource type, resource name, and resource details.
+
+    - To use the TiDB Cloud Dedicated network container resource, set the resource type as `tidbcloud_dedicated_network_container`.
+    - For the resource name, you can define it as needed, for example, `example`.
+    - If you do not know how to get the values of the required arguments, see [Set a CIDR for a Region](/tidb-cloud/set-up-vpc-peering-connections.md#prerequisite-set-a-cidr-for-a-region).
+    - For more information about the TiDB Cloud Dedicated network container specification, see [tidbcloud_dedicated_network_container (Resource)](https://registry.terraform.io/providers/tidbcloud/tidbcloud/latest/docs/resources/dedicated_network_container).
+
+3. Run the `terraform apply` command. It is not recommended to use `terraform apply --auto-approve` when you apply a resource.
+
+    ```shell
+    $ terraform apply
+
+    Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
+      + create
+
+    Terraform will perform the following actions:
+
+        # tidbcloud_dedicated_network_container.example will be created
+        + resource "tidbcloud_dedicated_network_container" "example" {
+            + cidr_notation        = "172.16.16.0/21"
+            + cloud_provider       = (known after apply)
+            + labels               = (known after apply)
+            + network_container_id = (known after apply)
+            + project_id           = "1372813089454543324"
+            + region_display_name  = (known after apply)
+            + region_id            = "aws-ap-northeast-2"
+            + state                = (known after apply)
+            + vpc_id               = (known after apply)
+        }
+
+    Plan: 1 to add, 0 to change, 0 to destroy.
+
+    Do you want to perform these actions?
+        Terraform will perform the actions described above.
+        Only 'yes' will be accepted to approve.
+
+        Enter a value:
+    ```
+
+   In the preceding result, Terraform generates an execution plan for you, which describes the actions Terraform will take:
+
+   - You can check the difference between the configurations and the states.
+   - You can also see the results of this `apply`. It will add a new resource, and no resource will be changed or destroyed.
+   - The `known after apply` shows that you will get the value after `apply`.
```

Comment:
```suggestion
    In the preceding result, Terraform generates an execution plan for you, which describes the actions Terraform will take:

    - You can check the differences between the configurations and the states.
    - You can also see the results of this `apply`. It will add a new resource, and no resource will be changed or destroyed.
    - The `known after apply` shows that you will get the value after `apply`.
```